src/player.cpp

```diff
@@ -157,7 +157,7 @@ static const itype_id OPTICAL_CLOAK_ITEM_ID( "optical_cloak" );
 static bool should_combine_bps( const player &, size_t, size_t );
 
 
-player::player() : Character()
```

Comment:
When I was comparing assignments in the constructors I found an inconsistency: `Character()` doesn't explicitly invoke the ancestor's constructor, but `player()` does. So I could either add the invocation to the `Character` or remove one from the `player`. I've chosen the latter, but I'm equally ok with the former.